Dockerfile-Dev

```diff
@@ -0,0 +1,3 @@
+FROM quay.io/pires/docker-jre:8u131_alpine3.6.2
+COPY ./target/hazelcast-kubernetes-bootstrapper-*-SNAPSHOT.jar /bootstrapper.jar
```

Comment:
For local development, yes and it is described in README.md
For kubernetes deployment, there will be another repo which has a Dockerfile and pulls the release artifacts from this repo. This is how the hazelcast-kubernetes image are currently built and I don't want to change that too much.